model/span_test.go

```diff
@@ -227,12 +227,12 @@ func TestIsFirehoseEnabled(t *testing.T) {
 func TestGetSpanKind(t *testing.T) {
 	span := makeSpan(model.String("sampler.type", "lowerbound"))
 	spanKind, found := span.GetSpanKind()
-	assert.Equal(t, "", spanKind)
+	assert.Equal(t, trace.SpanKind.String(0), spanKind)
```

Comment:
Using the predefined enum consts would be more readable:
- `trace.SpanKindUnspecified.String()`
- `trace.SpanKindClient.String()`

I think it's also okay to just leave it as `""` and `"client"` since it's better to decouple expected values from the thing that's being tested, and is the final displayed value in the span tag anyway.

Reply:
Thanks for this direction, I will revert it as it was.